pyOCD/board/mbed_board.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, name, target, binary):
                 "1600": BoardInfo(  "Bambino 210",          "lpc4330",          "l1_lpc4330.bin",       ),
                 "1605": BoardInfo(  "Bambino 210E",         "lpc4330",          "l1_lpc4330.bin",       ),
                 "2201": BoardInfo(  "WIZwik_W7500",         "w7500",            "l1_w7500mbed.bin",     ),
+                "7402": BoardInfo(  "mbed 6LoWPAN Border Router HAT",            "k64f",             "l1_k64f.bin",          ),
```

Comment:
Nitpick - can the spacing be adjusted so the ending bracked closes where the rest do?